src/runtime.js

```diff
@@ -140,6 +141,9 @@ export function adopt(global) {
   };
 
   /** @const */
+  global.AMP.config = config;
```

Comment:
It cannot hurt, although it currently falls under "do not obfuscate symbols not ending in _"